roles/efi_boot_mgr/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Validation for UEFI Boot Manager
+  ansible.builtin.assert:
+    that:
+      - ebm_nodes is defined
+      - ebm_nodes | length
+
+- name: Delete UEFI boot entries in nodes
+  vars:
+    _ebm_script: "{{ lookup('file', 'rm-efiboot') | ansible.builtin.b64encode }}"
+  ansible.builtin.shell: >
+    set -eo pipefail;
+    {{ ebm_oc_path }} debug node/{{ node_name }} --
+    bash -c "echo {{ _ebm_script }} | base64 -d > /host/tmp/rm-efiboot";
+    {{ ebm_oc_path }} debug node/{{ node_name }} -- chroot /host bash -x /tmp/rm-efiboot;
+    {{ ebm_oc_path }} debug node/{{ node_name }} -- rm -f /host/tmp/rm-efiboot;
```

Comment:
because this is a draft, I guess for now this is only prepared for connected clusters right? I wonder how to plug the image registry for disconnected envs in these debug commands.

Reply:
It is possible to invoke the toolbox which may be easier to mirror.

Reply:
>  because this is a draft, I guess for now this is only prepared for connected clusters right?

I wanted to complete some tests before moving it for review, at this point I've done connected *and* disconnected :)

>  I wonder how to plug the image registry for disconnected envs in these debug commands.

Good question, opposite to [sos_report](https://github.com/redhatci/ansible-collection-redhatci-ocp/tree/main/roles/sos_report)

Where an image is required because the provided debug image does not include sos_report tool:

https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/5fda7521298af599a7a61ea431d1340de8cae11b/roles/sos_report/README.md?plain=1#L15

>  It is possible to invoke the toolbox which may be easier to mirror.

We actually just need the `tools` image which is the default image used in an `ocm adm debug`, so we don't need to mirror it for this role.  It is already mirrored in a disconnected cluster because it is part of the OCP release images 

For example:

```Shell
❯ curl -sL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.16/release.txt | grep -P '^\s*tools'
  tools                                          quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:651732e89586140ee9fd7ca8772e0adbf9606fff461759235dbbd42af5636eae
```

It means that in connected, it will pull it from quay.io/openshift-release-dev/ocp-release and in disconnected the image is mirrored prior installation, so it will also be able to pull the right one in disconnected from the local registry as well.

hth